app-connector-client/server/routes/localApis.js

```diff
@@ -42,16 +69,16 @@ async function registerAll(req, res) {
     }
     catch (error) {
         var message = "There is an error while registering to kyma. Usually that is caused by an invalid or expired token URL."
-        LOGGER.error("Failed to connect to kyma cluster: %s", error)
+        LOGGER.error("Failed to connect to kyma cluster: %s", error.stack)

```

Comment:
Fixed in master branch already: it is not a connection problem but a registration problem. It should return a 500 if it is not caused by the user

Reply:
cool